### ISSUE TITLE
Prisma Cloud (née Twistlock) Security Vulnerabilities

### ISSUE THREAD

**Issue by nutjob4life** [EDRN/P5#104]
See the file `twistlock_registry_01_26_21_16_56_46.csv` for the image vulnerabilities in the 5.1.0 Docker image. The file is not attached to this issue due to its sensitive nature.

**Comment by nutjob4life** [comment]
Second round:


[twistlock_registry_02_22_21_23_11_21.csv.zip](https://github.com/EDRN/P5/files/6029154/twistlock_registry_02_22_21_23_11_21.csv.zip)

**Comment by nutjob4life** [comment]
Third and final round:

[twistlock_registry_02_23_21_22_57_52.csv.zip](https://github.com/EDRN/P5/files/6038114/twistlock_registry_02_23_21_22_57_52.csv.zip)

**Comment by nutjob4life** [comment]
Here are the arguments for not implementing mitigations for the remaining issues:


- CVE-2020-35457, CVE-2021-27218, CVE-2021-27219: these are for issues in glib (the GNOME Desktop UI), one of which is disputed. However, the application in question is not a GNOME Desktop application. The glib library is included as a dependency for indexing features and is not subject to the GUI issues indicated.
- CVE-2021-27212: this issue does not have a fix from the vendor. However, it is not an issue for the EDRN application as it affects only the "slapd" LDAP server. The application being scanned does not run "slapd" at all, nor is "slapd" even installed.
- GHSA-hg79-j56m-fxgv: this is a false positive for a vulnerability in React; however the EDRN application is not a React application (it's Plone).
- CVE-2018-1000620, CVE-2016-10540: these are flagged for side-effect dependencies that are not used by the EDRN application. Further, during image creation, we excise (delete) these dependencies so their detections are false positives.
- GHSA-hg79-j56m-fxgv, CVE-2015-9251: these are flagged for false positives for jQuery 1.12.4. The version included in the EDRN application is jQuery 3.2.1.
- GHSA-27gm-ghr9-4v95: the TinyMCE editor was available only to trusted users of the EDRN application, but as an extra precaution, we have disabled it.
- CVE-2020-11538, CVE-2020-35654, CVE-2020-35653, PRISMA-2021-0015, PRISMA-2021-0010, CVE-2020-10379: unfortunately the Python Imaging Library (or "Pillow") must remain at version 6.2.2 until we can upgrade the version of Python in the application's image. However, as a precaution, we have excised (deleted) the shared objects and Python support files that the issues referenced (SGI, PCX, Tiff image handling) since these features were never used by the EDRN application
- "Private key found in image": this is a testing key to ensure certain encryption features work and is not used by the application.

No word on if these are accepted but they usually are so I'm closing this issue.
